client/src/assets/components/posts/PostBody.jsx

```diff
@@ -1,26 +1,34 @@
 import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import ReactMarkdown from 'react-markdown';
 
 const PostBody = ({ body }) => {
     const [expandedPost, setExpandedPost] = useState(false);
 
+    // const renderBodyWithTags = (text) => {
+    //     return text.split(' ').map((word, i) => {
+    //         if (word.startsWith('#')) {
+    //             const cleanTag = word.replace(/[^\w#]/g, '').slice(1).toLowerCase();
+    //             return (
+    //                 <Link
+    //                     key={i}
+    //                     // to={`/tags/${cleanTag}`}
+    //                     to={`#`}
+    //                     className="text-primary hover:underline"
+    //                 >
+    //                     {word + ' '}
+    //                 </Link>
+    //             );
+    //         }
+    //         return <span key={i}>{word + ' '}</span>;
+    //     });
+    // };
+    
     const renderBodyWithTags = (text) => {
-        return text.split(' ').map((word, i) => {
-            if (word.startsWith('#')) {
-                const cleanTag = word.replace(/[^\w#]/g, '').slice(1).toLowerCase();
-                return (
-                    <Link
-                        key={i}
-                        // to={`/tags/${cleanTag}`}
-                        to={`#`}
-                        className="text-primary hover:underline"
-                    >
-                        {word + ' '}
-                    </Link>
-                );
-            }
-            return <span key={i}>{word + ' '}</span>;
-        });
+      return text.replace(/(^|\s)(#\w+)/g, (_, space, tag) => {
+        const tagText = tag.replace(/[^\w#]/g, '').slice(1).toLowerCase();
+        return `${space}[${tag}](/tags/${tagText})`;
```

Comment:
Can you comment this out for future use and create one with just a link with "#" so it doesn't forward to a page yet? We haven't implemented that feature just yet.